pyo3-macros-backend/src/method.rs

```diff
@@ -53,6 +53,7 @@ pub struct PyArg<'a> {
     pub ty: &'a syn::Type,
 }
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
Change to make Clippy happy

Reply:
We should consider addressing it. It seems that `RegularArg` is the big one here. We can probably put it or most of its fields behind a reference like the other variants.

Anyway that shouldn't block this PR, but please file an issue for it if you do not address it.

Reply:
>  We can probably put it or most of its fields behind a reference like the other variants.

Yes, but it does not seems trivial and will require some refactoring. I have open #5039 about it.
Anyway, this is a structure we do not keep in memory for a while, I am not sure how much we care about its size.